tests/set_test.go

```diff
@@ -21,3 +21,37 @@ func TestSet(t *testing.T) {
 		}
 	}
 }
+func TestSetWithNX(t *testing.T) {
+	conn := getLocalConnection()
+	for _, tcase := range []DTestCase{
+		{
+			InCmds: []string{"SET K V NX", "GET K"},
```

Comment:
Whetever keys you are planning to use in your test make sure to clean them up by calling deleteTestKeys utility

Reply:
yeah, sure

Reply:
Add negative tests too

like SET a key, then SET the key again using NX param